server/views/components/formFields/form-page-day-weeks-input/macro.njk

```diff
@@ -24,7 +24,7 @@
     {% if context.errors[fieldName] %}
       <p id="{{ fieldname }}-error" class="govuk-error-message" data-cy-error-{{ fieldname }}="true">
```

Comment:
Another `fieldname` with incorrect case here

Reply:
Ouch. I guess that's a result of copy'n'paste.  This shows that we're lacking integration testing on error handling.